docs/pages/database-access/guides/postgres-cloudsql.mdx

```diff
@@ -66,8 +66,14 @@ Teleport Database service with the following IAM roles:
 - "Service Account Token Creator": will allow Database service to generate
   IAM auth tokens when connecting to the database as the service account user
   we created above.
-- "Cloud SQL Viewer": will allow Database service to automatically download
-  your Cloud SQL instance root CA certificate.
+- Either "Cloud SQL Viewer" or "Cloud SQL Admin":
+  - "Cloud SQL Viewer" will allow Database service to automatically download
```

Comment:
```suggestion
  - "Cloud SQL Viewer" will allow the Database Service to automatically download
```

Reply:
This was copied from the previous release. There is another bullet above that has the same issue. I'll update everything.